### PR TITLE
Add a widget for the illustration setting

### PR DESCRIPTION
**Pull request by ranquild**
Epic https://github.com/metabase/metabase/issues/22826

Extract code from the Metabot widget and use it for the illustration widget.

How to test:
- Run Metabase EE
- Go to Admin -> Settings -> Appearance
- At the bottom of the page there should be a custom widget for the illustration setting
- Make sure the setting can be toggled on and off

Screenshot 2022-07-13 at 21 45 55

### CODE REVIEW

File: enterprise/frontend/src/metabase-enterprise/whitelabel/components/LighthouseToggleWidget/LighthouseToggleWidget.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import styled from "@emotion/styled";
+import { hueRotate } from "metabase/lib/colors";
+
+export const LighthouseImage = styled.div`
+  width: 99.5px;
```
Comment (WiNloSt) [inline enterprise/frontend/src/metabase-enterprise/whitelabel/components/LighthouseToggleWidget/LighthouseToggleWidget.styled.tsx:5]: Why not `100px`? Seeing a fraction seems a bit strange. I believed this is from down-sizing the Metabot image which is 1194 × 1080px and is scaled down to 99.5 x 90px, but we could use a different number for the lighthouse.

File: enterprise/frontend/src/metabase-enterprise/whitelabel/components/ImageToggle/ImageToggle.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+export const ToggleRoot = styled.div`
+  display: flex;
+  flex: 1 1 auto;
+  max-width: 33rem;
+  border: 1px solid ${color("border")};
+  border-radius: 0.5rem;
+`;
+
+export const ImageContainer = styled.div`
+  padding: 0.5rem 1rem 0.75rem 1.25rem;
```
Comment (WiNloSt) [inline enterprise/frontend/src/metabase-enterprise/whitelabel/components/ImageToggle/ImageToggle.styled.tsx:13]: Not sure I'd justify using this padding for the lighthouse image. I guess the intention behind this unequal padding is to make the Metabot look center.
image
image

I'd rather use more uniform padding and adjust the Metobot position separately.

**Review by WiNloSt — APPROVED** [review]

File: enterprise/frontend/src/metabase-enterprise/whitelabel/components/ImageToggle/ImageToggle.styled.tsx (hunk 2 above)
Comment (ranquild) [reply enterprise/frontend/src/metabase-enterprise/whitelabel/components/ImageToggle/ImageToggle.styled.tsx]: Agree, fixed

**Comment by mazameli** [comment]
Nice work!

**Pull request merged**